### PR TITLE
feat(scheduledevent): add support for event cover images

### PR DESCRIPTION
**Pull request by suneettipirneni**
**Please describe the changes this PR makes and why it should be merged:**

Adds `#image` and `#coverImageURL()` for getting the scheduled event cover image.

- https://github.com/discord/discord-api-docs/pull/4380

Depends on:
- https://github.com/discordjs/discord.js/pull/7298

**Status and versioning classification:**
- Code changes have been tested against the Discord API, or there are no code changes
- I know how to update typings and have done so, or typings don't need updating
- This PR changes the library's interface (methods or parameters added)

### CODE REVIEW

**Comment by suneettipirneni** [comment]
> /rest has landed.

Updated

**Review by iCrawl — APPROVED** [review]

**Review by SpaceEEC — APPROVED** [review]

File: packages/discord.js/src/structures/GuildScheduledEvent.js [hunk 1]
```diff
@@ -152,6 +152,21 @@ class GuildScheduledEvent extends Base {
...
+    /**
+     * The cover image hash for this scheduled event
+     * @type {?string}
+     */
+    this.image = data.image ?? null;
+  }
+
+  /**
+   * The URL of this scheduled event's cover image
+   * @param {BaseMakeURLOptions} [options={}] Options for image URL
```
Comment (vaporox) [inline packages/discord.js/src/structures/GuildScheduledEvent.js:165]: This type doesn't exist as a typedef

File: packages/discord.js/src/structures/GuildScheduledEvent.js [hunk 2]
```diff
@@ -152,6 +152,21 @@ class GuildScheduledEvent extends Base {
...
+    /**
+     * The cover image hash for this scheduled event
+     * @type {?string}
+     */
+    this.image = data.image ?? null;
+  }
+
+  /**
+   * The URL of this scheduled event's cover image
+   * @param {BaseImageURLOptions} [options={}] Options for image URL
```
Comment (vaporox) [inline packages/discord.js/src/structures/GuildScheduledEvent.js:165]: This type doesn't exist as a typedef

**Review by kyranet — APPROVED** [review]

**Pull request merged**